megamek/src/megamek/server/Server.java

```diff
@@ -37272,35 +35915,35 @@ private void handleAttacks(boolean pointblankShot) {
     }
 
     /**
-     * @return
+     * @return the current server instance
      */
     public static Server getServerInstance() {
         return serverInstance;
     }
 
     /**
-     * create a <code>SmokeCloud object and add it to the server list</code>
+     * create a <code>SmokeCloud</code> object and add it to the server list
      *
-     * @param coords
+     * @param coords the location to create the cloud
      */
-    public void createSmoke(Coords coords) {
+    public void createSmoke(Coords coords) { // TODO : Windchild is this needed? No usages
         createSmoke(coords, 1, 0);
     }
 
     /**
-     * create a <code>SmokeCloud object and add it to the server list</code>
+     * create a <code>SmokeCloud</code> object and add it to the server list
      *
-     * @param coords
-     * @param level  1=Light 2=Heavy Smoke
+     * @param coords the location to create the smoke
+     * @param level  1=Light 2=Heavy Smoke 3:light LI smoke 4: Heavy LI smoke
      */
-    public void createSmoke(Coords coords, int level) {
+    public void createSmoke(Coords coords, int level) { // TODO : Windchild is this needed? No usages
```

Comment:
Is this needed? No usages

Reply:
Unused versions of overloaded methods. They can go.